released/packages/coq-ceramist/coq-ceramist.1.0.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Coq library for reasoning about probabilistic algorithms"
+description: """
+  Ceramist extends coq-infotheo to support reasoning about probabilistic algorithms,
+  and includes a collection of lemmas on random oracle based hash functions.
+
+  Provides an example implementation of a bloom filter and uses the library to prove
+  the probability of a false positive.
+"""
+
+homepage: "https://github.com/certichain/ceramist"
+dev-repo: "git+https://github.com/certichain/ceramist.git"
+bug-reports: "https://github.com/certichain/ceramist/issues"
+maintainer: "kirang@comp.nus.edu.sg"
+authors: [
+  "Kiran Gopinathan"
+  "Ilya Sergey"
+]
+license: "GPL-3.0-or-later"
+depends: [
+  "coq" {>= "8.11.0" & < "8.11.1"}
```

Comment:
I guess `< 8.11.1` is due to https://github.com/coq/opam-coq-archive/issues/1230?

Reply:
Yes, I wasn't  able to install coq-finmap when using `coq.8.11.1`

Reply:
PR https://github.com/coq/opam-coq-archive/pull/1232 introduces `coq-mathcomp-finmap` 1.4.1 which fixes this issue and it should be available on opam within a few hours.